example/README.md

```diff
@@ -0,0 +1,39 @@
+# React Native Example App
+
+## Running
+
+```sh
+yarn install
+cd example
+yarn install
```

Comment:
Just `yarn` (or `yarn bootstrap`) in the root directory should be used to setup the repo/example app.